src/components/BoardsColumn.vue

```diff
@@ -1,57 +1,61 @@
-            </div>
+  <div class="columns-container" :class="{ 'columns-container--sizes': !logo }">
+    <div class="flex gap-6 h-full">
+      <div
+        v-for="(column, index) in columns"
+        :key="index"
+        class="flex flex-col"
+      >
+        <div class="flex items-center gap-2 mb-8 min-w-[280px]">
+          <div
+            class="h-[15px] w-[15px] rounded-full"
+            :class="circleColor(column)"
+          ></div>
+          <p class="text-xs text-gray-400 uppercase">
+            {{ column.name }} ({{ column.tasks?.length || 0 }})
+          </p>
         </div>
-        <Teleport  to="body">
-            <transition name="dialog">
-                <see-task-dialog
-                    v-if="clickedTitle != null"
-                    :showEditForm="showEditForm"
-                    :showDeleteForm="showDeleteForm"
-                    :closeSeeTask="closeSeeTask"
-                />
-            </transition>
-        </Teleport>
-        <transition name="dialog">
-            <confirmation-dialog
-                v-if="isDeleteTaskDialogShown"
-                elementToDelete="task"
-                elementName="Research pricing points of various competitors and trial different business models"
-                :closeDialog="() => isDeleteTaskDialogShown = false"
-            />
-        </transition>
-        <transition name="dialog">
-            <task-dialog
-                v-if="isEditTaskDialogShown"
-                action="edit"
-                :selectedMultiOptionItems="selectedMultiOptionItems"
-                :closeDialog="() => isEditTaskDialogShown = false"
-            />
-        </transition>
+        <task-card
+          @change="(title) => (clickedTitle = title)"
+          v-for="{ title } in column.tasks"
+          :key="title"
+          :howManyCompleted="0"
+          :howManySubtasks="0"
+          :title="title"
+          :isClickedTask="clickedTitle === title"
+        />
+      </div>
+      <div class="new-column group" tabindex="0">
+        <span class="new-column-text">+ New Column</span>
+      </div>
     </div>
+    <Teleport to="body">
+      <transition name="dialog">
+        <see-task-dialog
+          v-if="clickedTitle != null"
+          @close-dialog="closeSeeTask"
+          @show-edit-form="showEditForm"
+          @show-delete-form="showDeleteForm"
+        />
+      </transition>
+    </Teleport>
+    <transition name="dialog">
+      <confirmation-dialog
+        v-if="isDeleteTaskDialogShown"
+        @close-dialog="isDeleteTaskDialogShown = false"
+        elementToDelete="task"
+        elementName="Research pricing points of various competitors and trial different business models"
+      />
+    </transition>
+    <transition name="dialog">
+      <task-dialog
+        v-if="isEditTaskDialogShown"
+        @close-dialog="isEditTaskDialogShown = false"
+        action="edit"
+        :selectedMultiOptionItems="selectedMultiOptionItems"
+      />
```

Comment:
So if it's a `task-dialog` does it need an explicit `close-dialog` event? If so, then why just an `action`? I'd go for a `task-dialog` with a `close` event and an `action`